releases/nightly-build/compose-files/Makefile

```diff
@@ -0,0 +1,129 @@
+.PHONY: help portainer portainer-down run pull gen down clean remove
+.SILENT: help get-token del-token
+
+# TODO: When cutting next release change these to edgexfoundry & the actual Edgex release version
+REPOSITORY:=nexus3.edgexfoundry.org:10004
+CORE_EDGEX_VERSION:=master
+# Note: Device services, consul and App Services versions may differ from the Edgex release version
+#	    The versions for these services still have to be changed in the compose file as they all may be different.
+
+COMPOSE_FILES:=-f docker-compose-nexus-base.yml
+OPTIONS:=" arm64 no-secty no-ds mqtt dev ui " # Must have spaces around words for `filter-out` function to work properly
+
+ARGS:=$(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+$(eval $(ARGS):;@:)
+
+ifeq (dev, $(filter dev,$(ARGS)))
+	DEV:=-dev
+endif
+ifeq (arm64, $(filter arm64,$(ARGS)))
+	ARCH:=-arm64
+	REDIS_ARCH:=arm64v8/
+	KONG_UBUNTU:=-ubuntu
+endif
+ifneq (no-ds, $(filter no-ds,$(ARGS)))
+	COMPOSE_FILES:=${COMPOSE_FILES} -f docker-compose-nexus-add-device-services.yml
+endif
+ifeq (no-secty, $(filter no-secty,$(ARGS)))
+	SECURITY:="NO"
+else
+	SECURITY:="YES"
+	COMPOSE_FILES:=${COMPOSE_FILES} -f docker-compose-nexus-add-security.yml
+endif
+ifeq (mqtt, $(filter mqtt,$(ARGS)))
+	ifeq (YES,$(SECURITY))
+		COMPOSE_FILES:=${COMPOSE_FILES} -f docker-compose-nexus-add-mqtt.yml -f docker-compose-nexus-add-security.yml
+	else
+		COMPOSE_FILES:=${COMPOSE_FILES} -f docker-compose-nexus-add-mqtt.yml
+	endif
+endif
+ifeq (ui, $(filter ui,$(ARGS)))
+	COMPOSE_FILES:=-f docker-compose-nexus-ui.yml
+endif
+SERVICES:=$(filter-out $(OPTIONS),$(ARGS))
+USER:=$(wordlist 1,$(words $(SERVICES)),$(SERVICES)) # for use on get-token and del-token
+
+help:
+	echo "See README.md in this folder"
+
+portainer:
+	docker-compose -p portainer -f docker-compose-portainer.yml up -d
+
+portainer-down:
+	docker-compose -p portainer -f docker-compose-portainer.yml down
+
+run:
```

Comment:
This causes the images to be pulled every time I run, which I don't want as I bring the stack up and down many times during a testing and debug session. Also, not something needed when these files are pushed to a release folder since the images don't changes like they are for our `master` tags.

Reply:
I did make the change to use `gen` and then simply run the generated compose file. See latest changes.